src/Uno.UI.FluentTheme.v1/AssemblyInfo.cs

```diff
@@ -8,13 +8,14 @@
 [assembly: InternalsVisibleTo("Uno.Wasm")]
 [assembly: InternalsVisibleTo("Uno.UI.Tests")]
 
-#if NET6_0_OR_GREATER
-[assembly: System.Reflection.AssemblyMetadata("IsTrimmable", "True")]
-#elif __IOS__
+[assembly: AssemblyMetadata("IsTrimmable", "True")]
```

Comment:
This attribute is ignored by the linker in the legacy SDK (before net6)
IOW it does not matter (much) if it stays under `NET6_0_OR_GREATER` or not

Reply:
We need this to be enabled for the netstandard2.0 target (for skia+gtk), this is why it's forcibly added for all targets.